eclipse-collections-api/src/main/java/org/eclipse/collections/api/multimap/bag/MutableBagIterableMultimap.java

```diff
@@ -35,6 +35,24 @@ default MutableBagIterableMultimap<K, V> withKeyValue(K key, V value)
         return this;
     }
 
+    /**
+     * Puts the key / values combination into the MutableBagIterableMultimap and returns
+     * the MutableBagIterableMultimap (this).
+     * @since 11.1
+     */
+    @Override
+    default MutableBagIterableMultimap<K, V> withKeyMultiValues(K key, V... values)
+    {
+        if (values != null && values.length > 0)
+        {
+            for (V value : values)
+            {
+                this.put(key, value);
+            }
+        }
+        return this;
```

Comment:
In overrides, you could have a similar implementation just like the one I suggested above. Another alternative is to call super and cast.

```java
return (MutableBagIterableMultimap<K, V>) MutableMultimap.super.withKeyMultiValues(key, values);
```

Reply:
Addressed.